crates/vm/src/arch/extensions.rs

```diff
@@ -457,6 +460,14 @@ impl<F: PrimeField32> SystemBase<F> {
         PROGRAM_BUS
     }
 
+    pub fn memory_bridge(&self) -> MemoryBridge {
+        self.memory_controller.memory_bridge()
+    }
+
+    pub fn offline_memory(&self) -> Arc<Mutex<OfflineMemory<F>>> {
```

Comment:
add comment around how this should only be used when passing into a chip constructor